lib/sgcPrompt.js

```diff
@@ -26,11 +26,18 @@ const sgcPrompt = () => {
   }, {
     type: 'input',
     name: 'description',
-    message: 'Describe your commit message:',
+    message: 'Enter your commit message:',
+    validate: input => new Promise((resolve, reject) => {
+      if (input) {
+        resolve(true);
+      } else {
+        reject('A commit message is mandatory!');
+      }
+    }),
```

Comment:
AFAIK validate doesn't need a promise as long as it's not async. In this case I would prefer:

```js
validate: input => input ? true : 'A commit message is mandatory!',
```